src/main/java/pl/wedding/management/service/weddinghall/model/WeddingHall.java

```diff
@@ -0,0 +1,15 @@
+package pl.wedding.management.service.weddinghall.model;
+
+import lombok.*;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+@Getter
+@NoArgsConstructor
```

Comment:
NoArgsConstructor has no sense. You receive empty object, with null fields, and won't be able to set any (No setter).
You can use @value designed for immutability, and please add @builder. It simplifies creating objects.